### PR TITLE
Shows the shiny streak when it is reset due to catching the shiny.

### PR DESCRIPTION
**Pull request by WitherredAway**
The thingy request in issue #257 
E.g.
Congratulations Witherr🖤! You caught a level 1 Ralts!

Shiny streak reset. (**652**)

These colors seem unusual.. ✨

### CODE REVIEW

File: cogs/spawning.py [hunk 1]
```diff
@@ -422,7 +422,7 @@ async def catch(self, ctx, *, guess: str):
 
         if member.shiny_hunt == species.dex_number:
             if shiny:
-                message += f"\n\nShiny streak reset."
+                message += f"\n\nShiny streak reset. (**{member.shiny_streak}**)"
```
Comment (michaelsong4399) [inline cogs/spawning.py:425]: We probably still need to add 1 to the displayed shiny streak as done in L428.

**Comment by WitherredAway** [comment]
OH right!

**Comment by WitherredAway** [comment]
thanks micool

**Comment by oliver-ni** [comment]
I question whether it would be accurate to add one or not.

**Comment by WitherredAway** [comment]
hm, since this catch also contributes to the streak (also the shiny catch) I'd imagine it would also +1 the streak since it is another catch of the same sh pokemon? because it doesn't +1 to the streak *after* the streak is reset (the streak is back to 0 after the shiny catch, so if it didn't +1 to the streak before it is reset and it also doesn't +1 to the streak after the reset, that one catch is just discarded for either streak?)

**Comment by WitherredAway** [comment]
idk if that made sense

**Pull request merged**